docker/docker-compose.yml

```diff
@@ -43,6 +43,8 @@ services:
       POSTGRES_DB: takahe
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: insecure_password
+    ports:
+      - "5432:5432"
```

Comment:
Keep this in your local version please, don't need port conflicts when postgres is running on the hos.

Reply:
How about I switch it to `5433`? That port is pretty much unused.

 It is quite useful to have this exposed as a default so I don't need to modify a in-tree file all the time.

Reply:
It's going to clash for someone I'm sure, but let's stick it in there and see if anyone complains.